mypyc/lib-rt/module_shim.tmpl

```diff
@@ -5,8 +5,10 @@ PyInit_{modname}(void)
 {{
     PyObject *tmp;
     if (!(tmp = PyImport_ImportModule("{libname}"))) return NULL;
+    PyObject *capsule = PyObject_GetAttrString(tmp, "init_{full_modname}");
```

Comment:
Check for null return. Otherwise we may decref a null pointer below.

Reply:
Fixed it. Thanks.